index.html

```diff
@@ -0,0 +1,149 @@
+          <a href="./index.html">
+            <img
+              src="./imges/icons/main-logo.svg"
+              alt="header-icon"
+              class="logo"
+            />
+          </a>
+          <ul>
+            <li>
+              <a href="about.html"> <span>About</span></a>
+            </li>
+            <li><a href="./index.html">program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponser</a></li>
+            <li><a href="#">News</a></li>
+            <li>
+              <h4 class="box">camping</h4>
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <div class="cover-cont">
+        <h1>
+          "hello! sharing world!" <br />
+          <span>
+            creatve commons <br />
+            global summit <br />
+            2021
+          </span>
+        </h1>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet
+          recusandae quia et? Ut cumque ullam magni Lorem ipsum dolor sit amet
+          consectetur adipisicing elit. Amet recusandae quia et? Ut cumque ullam
+          magni
+        </p>
+        <h3>2021.10.15</h3>
+        <h6>@ Lorem ipsum dolor sit amet consectetur</h6>
+      </div>
+    </section>
+    <section class="programs">
+      <h2>main programs</h2>
+      <hr class="main-program-line" />
+      <ul>
+        <li>
+          <img
+            src="./imges/icons/progrem-text-icon.svg"
+            alt=""
+            class="program-logo"
+          />
+          <h3>lecture</h3>
+          <p>
+            A lecture is an oral presentation intended to present information or
+            teach people
+          </p>
+        </li>
+        <li>
+          <img src="./imges/icons/ex.svg" alt="" class="program-logo" />
+          <h3>Exhibition</h3>
+          <p>
+            An exhibition, in the most general sense, is an organized
+            presentation and display
+          </p>
+        </li>
+        <li>
+          <img
+            src="./imges/icons/progrem-text-icon.svg"
+            alt=""
+            class="program-logo"
+          />
+          <h3>form</h3>
+          <p>
+            An exhibition, in the most general sense, is an organized
+            presentation and display
+          </p>
+        </li>
+        <li>
+          <img src="./imges/icons/workshop.svg" alt="" class="program-logo" />
+          <h3>workshop</h3>
+          <p>
+            a meeting at which a group of people engage in intensive discussion.
+          </p>
+        </li>
+        <li>
+          <img src="./imges/icons/sun.svg" alt="" class="program-logo" />
+          <h3>ignit</h3>
+          <p>
+            adjective. Slang for the word ignorant, Compare to: Silly, Retarted
+          </p>
+        </li>
+      </ul>
+      <h4><a>join global summit 2021</a></h4>
+    </section>
+    <h1 class="speaker-header">speakers</h1>
+    <hr />
+    <section class="speakers"></section>
```

Comment:
it's very hard to find photos of the same sizes on the internet it will take me a lot of time and it's not really a programming issue please let me know if I have to 
and I don't understand what you mean by (In mobile view the images are adjacent to their text and not above the text.)
and for the text, I thought that is my choice as long it the same font and color

Reply:
I guess you can resize the images in css. Just assign classes to the items in js and then style them in css.
In as much as it is not a programming issue the goal of this project is to replicate the design as much as possible but the content inside can change. This means the font size, color, image sizes, and layout, in general, should all be the same.

In the mobile view the images and the text should be like this:
![image](https://user-images.githubusercontent.com/68167810/124232268-1d2c3080-db1a-11eb-9cb1-5b41b063cdc9.png)